cmd/dusk/action.go

```diff
@@ -111,7 +131,7 @@ func setupProfiles(r *rpcbus.RPCBus) *diagnostics.ProfileSet {
 
 			p := diagnostics.NewProfile(i.Name, i.Interval, i.Duration, i.Start)
 			if err := s.Spawn(p); err != nil {
-				log.Panicf("Profiling task error: %s", err.Error())
+				log.WithError(err).Panic("Profiling task error")
```

Comment:
All these improper log entries can be changed  by a separate issue where we translate `log.*f(..., err)` to `log.WithError(err)` as per the guideline.

Reply:
Agree, I'll raise a ticket to continue the changes. But keeping it in a new file does not make sense, so I updated already. Are you suggesting to revert it ?

Reply:
No need to revert, just a suggestion for a new issue.